conjure_python_client/_http/requests_client.py

```diff
@@ -30,19 +30,17 @@
 T = TypeVar("T")
 # https://testssl.sh/openssl-rfc.mappping.html
 CIPHERS = (
-    "ECDHE-RSA-AES256-SHA384:"
-    "ECDHE-RSA-AES128-SHA256:"
-    "ECDH-RSA-AES256-SHA384:"
-    "ECDH-RSA-AES128-SHA256:"
-    "AES128-SHA256:"
-    "AES256-SHA256:"
-    "ECDHE-RSA-AES256-SHA:"
-    "ECDHE-RSA-AES128-SHA:"
-    "ECDH-RSA-AES256-SHA:"
-    "ECDH-RSA-AES128-SHA256:"
-    "AES256-SHA:"
-    "AES128-SHA:"
-    "TLS_FALLBACK_SCSV"
+    "TLS_AES_128_GCM_SHA256"
+    "TLS_AES_256_GCM_SHA384"
+    "TLS_CHACHA20_POLY1305_SHA256"
+    "ECDHE-ECDSA-AES128-GCM-SHA256:"
+    "ECDHE-RSA-AES128-GCM-SHA256:"
+    "ECDHE-ECDSA-AES256-GCM-SHA384:"
+    "ECDHE-RSA-AES256-GCM-SHA384:"
+    "ECDHE-ECDSA-CHACHA20-POLY1305:"
```

Comment:
https://testssl.sh/openssl-iana.mapping.html doesn't contain this one - instead it has ECDHE-ECDSA-CHACHA20-POLY1305-OLD?

Reply:
think actually this table is just outdated